app/scripts/DataFetcher.js

```diff
@@ -219,7 +219,7 @@ export default class DataFetcher {
    * Extract a slice from a matrix at a given position.
    *
    * @param {array} inputData: An array containing a matrix stored row-wise
-   * @param {array} arrayShape: The shape of the array, should be a two element array e.g. [256,256].
+   * @param {array} arrayShape: Shape of the array; should be a two element array e.g. [256,256].
```

Comment:
I never did know where to use semicolons when writing in English.

Reply:
I always thought of it as being a longer pause / stronger separation than a comma but less then a dot. 🤔 (Sorry this is a random comment)